core/modules/wmgr/python/config.py

```diff
@@ -122,21 +121,16 @@ def privDbEngine(self):
         inst = getEngineFromArgs(**kwargs)
         return inst
 
-    def proxyConn(self):
-        """
-        Return mysql connection object for proxy.
-
-        Sqlalchemy drivers have trouble with qserv, so we have to limit
-        ourself to direct mysqldb.
-        """
+    def proxyDbEngine(self):
+        """ Return database engine for proxy """
         kwargs = {}
         if self.proxyHost: kwargs['host'] = self.proxyHost
         if self.proxyPort: kwargs['port'] = self.proxyPort
-        if self.proxyUser: kwargs['user'] = self.proxyUser
+        if self.proxyUser: kwargs['username'] = self.proxyUser
         _log.debug('creating new connection %s', kwargs)
-        if self.proxyPasswd: kwargs['passwd'] = self.proxyPasswd
-        conn = sql.connect(**kwargs)
-        return conn
+        if self.proxyPasswd: kwargs['password'] = self.proxyPasswd
+        inst = getEngineFromArgs(**kwargs)
+        return inst
```

Comment:
I'll add "(password not shown)" to the message and remove intermediate "inst" variable.